static/js/util/validation.js

```diff
@@ -294,14 +294,16 @@ export function validateYear(input: string|number|null): Maybe<number> {
   if (year === undefined) {
     return Nothing();
   }
-  if ( year < 1800 ) {
+  let now = moment().year();
+  let cutoff = now - 120;
```

Comment:
can you put the `120` value into constants.js and refer to it here and in the unit tests?